lighthouse/blueprints/plates.py

```diff
@@ -136,18 +151,28 @@ def find_plate_from_barcode() -> FlaskResponse:
         barcodes_arg = request.args.get("barcodes")
         barcodes_list = barcodes_arg.split(",") if barcodes_arg else []
 
-        if len(barcodes_list) == 0:
-            return bad_request("Please include a barcode list")
+        assert len(barcodes_list) > 0, "Include a list of barcodes separated by commas (,) in the request"
+
+        plate_type = request.args.get(ARG_TYPE)
```

Comment:
We have to add the default, is it like that?
```suggestion
        plate_type = request.args.get(ARG_TYPE, ARG_TYPE_SOURCE)
```

Reply:
Great!